src/sentry/static/sentry/app/views/projectPlugins/organizationIntegrations.jsx

```diff
@@ -0,0 +1,71 @@
+import {Box} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from '../../locale';
+import AsyncComponent from '../../components/asyncComponent';
+import Link from '../../components/link';
+import Panel from '../settings/components/panel';
+import PanelBody from '../settings/components/panelBody';
+import PanelHeader from '../settings/components/panelHeader';
+import PluginIcon from '../../plugins/components/pluginIcon';
+import PanelItem from '../settings/components/panelItem';
+import theme from '../../utils/theme';
+
+const ProviderName = styled.div`
+  font-size: 16px;
+  font-weight: bold;
+  margin-bottom: 3px;
+`;
+
+const TeamName = styled.div`
+  color: ${p => p.theme.gray2};
+  font-size: 14px;
```

Comment:
yeah this stuff is way too manual, is it possible to pull it off theme?

Reply:
(not your fault)

Reply:
Probably more of a question for @ckj / @Chrissy / @billyvg

Reply:
GREAT QUESTION! 

tldr; right now it's kind of anything goes, so do whatever.

In the future, we should probably have some kind of type hierarchy system in `theme.js`, which we can also use for sizing up elements and things. 

As for the avatar and padding and stuff, it's generally a good idea to write those values in some kind of relative unit (percentages, vws, ems) or to use a flex layout to achieve the look you want. That way, the component is more fluid and can be resized with a top-down approach rather than having to go in and mutate a bunch of pixel values on props. But this is all kind of later stuff and shouldn't block this pull.